formatting.css

```
@@ -0,0 +1,184 @@
+h1{
+  color: #333333;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-shrink: 0;
+}
```

It's easier to read when we leave an empty line between two declarations.
```suggestion
}

```